tests/gis_tests/utils.py

```diff
@@ -1,13 +1,30 @@
-from unittest import skip
+import unittest
+from functools import wraps
 
 from django.conf import settings
-from django.db import DEFAULT_DB_ALIAS
+from django.db import DEFAULT_DB_ALIAS, connection
+
+
+def skipUnlessGISLookup(*gis_lookups):
+    """
+    Skip a test unless a database supports all of gis_lookups.
+    """
+    def decorator(test_func):
+        @wraps(test_func)
+        def skip_wrapper(*args, **kwargs):
+            if any(key not in connection.ops.gis_operators for key in gis_lookups):
+                raise unittest.SkipTest(
+                    "Database doesn't support the lookup: %s" % ", ".join(gis_lookups)
```

Comment:
I guess this could say "doesn't support all the lookups" in case gis_lookups is a list.